docs/ARCHITECTURE.md

```
@@ -0,0 +1,305 @@
+    └── [slug].astro     → /blog/:slug
+```
+
+### 2. コンポーネント駆動開発
+
+再利用可能なコンポーネントを `src/components/` に配置し、複数のページで使用します。
+
+```astro
+---
+// src/components/Button.astro
+interface Props {
+	text: string;
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)